ngo_design.css

```diff
@@ -0,0 +1,1315 @@
+@media (min-width: 480px) and (max-width: 800px){
+    body{
+        margin:0px;
+    }
+.flex-head{
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: center;
+     background-color: #333333;
+    color: white;
+   height: 70px;
+}
+.first_half, .sec_half{
+    padding-top: 2px;;
+    background-color:#333333;
+}
+.sec_half{
+    width:300px;
+}
+.border_bott{
+    padding-right: 20px;
+    padding-left: 60px;
+}
+.right_icons{
+    font-size: 24px;
+    padding-right: 20px;;
+}
+.left-icon{float: left;}
+
+.phone, .env{
+    font-size: 20px;
+    padding-right: 3px;
+}
+.flex-menu{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    background-color: #f1f1f1;
+}
+.main_menu{
+    width: 230px;
+    float:left;
+}
+.main_menu_right{
+    float:right;
+    width: 230px;;
+    font-size: 40px;
+}
+.fa-bars{float: right; 
+    padding-right: 20px; 
+    color: gray;
+    padding-top: 10px;
+}
+.cmp_name{
+    font-family: sans-serif;
+    font-size: 23px;
+    margin-top: 15px;
+    color: orangered;
+    padding-right: 10px;
+    padding-left: 20px;
+}
+.flex-container > div{
+    display: none;
+}
+.main_img{
+    
+    filter: brightness(40%);
+    height:200px;
+}
+.text-block{
+    position: absolute;
+    bottom: 20px;
+    right: 20px;
+    color: white;
+    top: 31%;
+    left: 0%; 
+    height: 100px;
+ }
+ .un_line{
+    border-bottom: 2px solid orangered;
+    padding-bottom:5px;
+}
+    
+.fset_1{
+    border-bottom: none;
+    border-top: none;
+    border-left: none;
+    border-right: none;
```

Comment:
The above 4 style can be combined to one
```
border: none
```

Reply:
changed the style definition to border:none;